### ISSUE TITLE
error: string is not a member of std

### ISSUE THREAD

**Issue by acxz** [ros-noetic-arch/ros-noetic-libfranka#2]
```
In file included from /home/acxz/.cache/yay/ros-noetic-libfranka/src/libfranka-release-upstream-0.7.1/src/control_tools.cpp:3:
/home/acxz/.cache/yay/ros-noetic-libfranka/src/libfranka-release-upstream-0.7.1/include/franka/control_tools.h:74:54: error: ‘string’ is not a member of ‘std’
   74 | bool setCurrentThreadToHighestSchedulerPriority(std::string* error_message);
      |                                                      ^~~~~~
/home/acxz/.cache/yay/ros-noetic-libfranka/src/libfranka-release-upstream-0.7.1/include/franka/control_tools.h:7:1: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
    6 | #include <cmath>
  +++ |+#include <string>
    7 | 
/home/acxz/.cache/yay/ros-noetic-libfranka/src/libfranka-release-upstream-0.7.1/include/franka/control_tools.h:74:62: error: ‘error_message’ was not declared in this scope
   74 | bool setCurrentThreadToHighestSchedulerPriority(std::string* error_message);
      |                                                              ^~~~~~~~~~~~~
/home/acxz/.cache/yay/ros-noetic-libfranka/src/libfranka-release-upstream-0.7.1/src/control_tools.cpp:29:75: error: ‘bool franka::setCurrentThreadToHighestSchedulerPriority(std::string*)’ redeclared as different kind of entity
   29 | bool setCurrentThreadToHighestSchedulerPriority(std::string* error_message) {
      |                                                                           ^
In file included from /home/acxz/.cache/yay/ros-noetic-libfranka/src/libfranka-release-upstream-0.7.1/src/control_tools.cpp:3:
/home/acxz/.cache/yay/ros-noetic-libfranka/src/libfranka-release-upstream-0.7.1/include/franka/control_tools.h:74:6: note: previous declaration ‘bool franka::setCurrentThreadToHighestSchedulerPriority’
   74 | bool setCurrentThreadToHighestSchedulerPriority(std::string* error_message);
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/franka.dir/build.make:95: CMakeFiles/franka.dir/src/control_tools.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:238: CMakeFiles/franka.dir/all] Error 2
make: *** [Makefile:182: all] Error 2
```

**Comment by acxz** [comment]
patched with https://github.com/frankaemika/libfranka/pull/66

**Comment by acxz** [comment]
closed with b9354b0
